website-next/styles/globals.css

```diff
@@ -23,42 +23,30 @@ a {
 }
 
 @font-face {
-  font-family: 'Marfa-Medium';
-  src: url('/fonts/Marfa-Semibold.otf');
+  font-family: 'RecklessNeue-Book';
+  src: url('/fonts/RecklessNeue-Book.woff2');
```

Comment:
what's with the woff vs woff2?

Reply:
woff2 is more encrypted. I understand doesn't deter the _determined_ font pirate, but probably prevents opportunistic typographic theft